src/medCore/database/medDatabaseController.cpp

```diff
@@ -587,13 +592,6 @@ bool medDatabaseController::moveDatabase( QString newLocation)
             qDebug() << "Restarting connection...";
             this->createConnection();
         }
-
-        // now delete the old archive
-        if(medStorage::removeDir(oldLocation))
```

Comment:
Yep, that would sound logical to me : if you're creating a new folder, then you wanted to move the database. If not, that means you want to reuse an older one and therefore keep the old one as well. But I'm open to discussion on this. In terms of code, it's a very easy change